### PR TITLE
Unevaluated watches handling fixes

### PR DESCRIPTION
**Pull request by johnny-keker**
* Fixed unevaluated watches painting
* Gray out all watches before debug session start
* Properly gray out watches if debug session was unsuccessfull

### CODE REVIEW

File: VSRAD.Package/DebugVisualizer/VisualizerTable.cs [hunk 1]
```diff
@@ -302,19 +308,29 @@ private static void LockWatchRowForEditing(DataGridViewRow row, bool canBeRemove
 
         #region Styling
 
-        public void GrayOutColumns(uint groupSize)
+        public void GrayOutColumns()
```
Comment (timlathy) [inline VSRAD.Package/DebugVisualizer/VisualizerTable.cs:311]: The name should be changed too (`GrayOutRows`?)

File: VSRAD.Package/DebugVisualizer/VisualizerTable.cs [hunk 2]
```diff
@@ -346,8 +362,9 @@ public void ApplyDataStyling(ProjectOptions options, uint groupSize, Server.Watc
                 _fontAndColor.FontAndColorState);
             columnStyling.Apply(_state.DataColumns);
 
+
```
Comment (timlathy) [inline VSRAD.Package/DebugVisualizer/VisualizerTable.cs:365]: Formatting

File: VSRAD.Package/DebugVisualizer/VisualizerTable.cs [hunk 3]
```diff
@@ -302,19 +308,29 @@ private static void LockWatchRowForEditing(DataGridViewRow row, bool canBeRemove
 
         #region Styling
 
-        public void GrayOutColumns(uint groupSize)
+        public void GrayOutColumns()
         {
-            _computedStyling.GrayOutColumns(groupSize);
```
Comment (timlathy) [inline VSRAD.Package/DebugVisualizer/VisualizerTable.cs:307]: This looks like the only usage of `ComputedColumnStyling.GrayOutColumns`, you can remove it. I don't think `ColumnStyling.GrayOutColumns` is used anywhere either, that can be removed too.

File: VSRAD.Package/DebugVisualizer/VisualizerTable.cs (hunk 1 above)
Comment (johnny-keker) [reply VSRAD.Package/DebugVisualizer/VisualizerTable.cs]: Thanks! Resolved in c1dce41

File: VSRAD.Package/DebugVisualizer/VisualizerTable.cs (hunk 2 above)
Comment (johnny-keker) [reply VSRAD.Package/DebugVisualizer/VisualizerTable.cs]: Thanks! Resolved in c1dce41

File: VSRAD.Package/DebugVisualizer/VisualizerTable.cs (hunk 3 above)
Comment (johnny-keker) [reply VSRAD.Package/DebugVisualizer/VisualizerTable.cs:307]: Thanks! Resolved in c1dce41

**Comment by johnny-keker** [comment]
> 
> 
> The last line is initially white, but if you change the group size, it will be colored gray. If you add a new watch after that, then it will be colored white again.

Thanks, fixed in d71e1d2

**Pull request merged**